GOT-StudentVersion/GOT-StudentVersion/Controllers/ViewController.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  ViewController.swift
+//  GOT-StudentVersion
+//
+//  Created by C4Q  on 11/2/17.
+//  Copyright © 2017 C4Q . All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+   
+    
+    // to access static properties, use its' class
+    @IBOutlet weak var GOTTableView: UITableView!
+    
+    
+    var arrOfSeason = GOTEpisode.seasons {
```

Comment:
Great job making this the main data source. It allows you to organize everything by the seasons.